src/test/java/org/zalando/intellij/swagger/completion/field/completion/swagger/SwaggerCompletionTest.java

```diff
@@ -178,6 +178,15 @@ public void thatItemsKeysInDefinitionsAreSuggested() {
                 .isOfSize(30);
     }
 
+    @Test
+    public void thatItemsKeysInParameterDefinitionsAreSuggested() {
+        getCaretCompletions("items_in_parameter_definitions")
+                .assertContains("type", "format", "items", "collectionFormat", "default", "maximum",
+                        "exclusiveMaximum", "minimum", "exclusiveMinimum", "maxLength", "minLength", "pattern",
+                        "maxItems", "minItems", "uniqueItems", "multipleOf")
+                .isOfSize(17);
```

Comment:
There are 16 completions items that have to be present and a check for the total number (17). Maybe including the 17th item and removing the check for the total number would make the test more robust.

Reply:
Nice catch, fixed!